OpenRA.Game/MiniYaml.cs

```diff
@@ -83,19 +83,19 @@ public override string ToString()
 
 	public class MiniYaml
 	{
-		static Func<string, string> StringIdentity = s => s;
-		static Func<MiniYaml, MiniYaml> MiniYamlIdentity = my => my;
+		static Func<string, string> stringIdentity = s => s;
+		static Func<MiniYaml, MiniYaml> miniYamlIdentity = my => my;
```

Comment:
I think these would both be better as readonly (requiring uppercase names again).